HHS/Head-Start-TTADP
# [TTAHUB-725] Activity Summary Page Redesign

## Description of change

Redesign for the activity summary page. Changes include using an asterisk '*' required label, '- Select -' place holder text, label re-wording, and adding context.

**UPDATE 04/12:**

Some last minute changes where added.

- Updated validation messages to match validation mockup on jira.
- Required input boxes are no highlighted in red.
- The Save and Back buttons at the bottom are now in a different order.

## How to test

Create a new report make sure that it matches the mockup linked to TTAHUB-725. The context section should now appear at the bottom of the page. 

- Creating / Submitting / Approving a report should work as it did before.
- Viewing the report in review should show the context section in the activity summary.
- Viewing an approved report should show context in the summary section.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-725


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
